docs/api_specs/includes/watcher_api_specs.md

```diff
@@ -0,0 +1,314 @@
+    }
+}
+```
+
+Gets exit data for a given utxo
+
+### HTTP Request
+
+`POST /utxo.get_exit_data`
+
+### JSON Body
+
+Key | Type | Description
+--------- | ------- | -----------
+utxo_pos | Integer | Utxo position (encoded as single integer, the way contract represents them)
+
+
+
+## Transaction - Submit
+
+```shell
+http POST /transaction.submit transaction=f8d083015ba98080808080940000...
+```
+
+```elixir
+// TODO
+```
+
+```javascript
+// TODO
+```
+
+> The above command returns JSON document:
+<aside class="warning">Is it OK? Following is planned child chain response for corresponding endpoint</aside> 
+
+```json
+{
+      "version": "1",
+      "success": true,
+      "data": { 
+          "blknum": 123000,
+          "txindex": 111,
+          "txhash": "bdf562c24ace032176e27621073df58ce1c6f65de3b5932343b70ba03c72132d"
+      }
+}
+```
+
+Watcher passes signed transaction to the child chain only if it's secure (better explaination needed)
+
+### HTTP Request
+
+`POST /transaction.submit`
+
+### JSON Body
+
+Key | Type | Description
+--------- | ------- | -----------
+transaction | Hex encoded string | Signed transaction RLP-encoded to bytes and HEX-encoded to string
+
+
+
+
+## Status
+
+```shell
+http POST /status
+```
+
+```elixir
+// TODO
+```
+
+```javascript
+// TODO
+```
+
+> The above command returns JSON document:
+
+```json
+{
+    "version": "1",
+    "success": true,
+    "data": {
+        "last_validated_child_block_number": 10000,
+        "last_mined_child_block_timestamp": 1535031020,
+        "last_mined_child_block_number": 11000,
+        "eth_syncing": true,
+        "byzantine_events":
+        [
+            {
+                "event": "invalid_exit",
+                "details": {
+                    "eth_height"  : 100,
+                    "utxo_pos"  : 100,
+                    "owner"  : "B3256026863EB6AE5B06FA396AB09069784EA8EA",
+                    "currency"  : "0000000000000000000000000000000000000000",
+                    "amount" : 100
+                }
+            }
+        ]
+    }
+}
+```
+
+Gets plasma network and Watcher status
+
+### HTTP Request
+
+`POST /status`
+
+### JSON Body
+
+No parameters are required.
+
+<aside class="warning">
+The most critical function of the Watcher is to monitor the ChildChain and report dishonest activity. 
+The user must call the `/status` endpoint periodically to check. Any situation that requires the user to either exit or challenge an invalid exit will be included in the `byzantine_events` field.
+</aside>
+
+### Byzantine events
+All of the following events indicate a byzantine chain and that the user should either exit or challenge.
+
+#### `invalid_exit`
```

Comment:
Is this really a 'byzantine' event?
My understanding of 'byzantine' is that the chain has been compromised somehow (most likely due to a dishonest operator) and everyone should exit. 
An `invalid_exit` though could just be a dishonest user, and as long as it's challenged the chain itself is fine.

Reply:
I'd say yes. The actor (user) is acting byzantine in the sense "not following protocol"/"arbitrarily".

I think it falls in the broad category of "byzantine" behavior, whereas what you seem to have in mind is "byzantine child chain". I tried to follow this differentiation or otherwise be clear about what's going on, but might have missed some spots.

Reply:
Got it, thanks 👍